src/flash/com/longtailvideo/jwplayer/media/VideoMediaProvider.as

```diff
@@ -97,9 +99,10 @@ public class VideoMediaProvider extends MediaProvider {
     /** Load new media file; only requested once per item. **/
     override public function load(itm:PlaylistItem):void {
         setState(PlayerState.LOADING);
-        if (_item !== itm) {
+        if (_item !== itm || _forceReload) {
```

Comment:
complete() sets the state to IDLE so:

`if (_item !== itm || state === IDLE) {`